dem_stitcher/geoid.py

```diff
@@ -11,7 +11,7 @@
 from .rio_tools import reproject_arr_to_match_profile, translate_profile
 from .rio_window import read_raster_from_window
 
-AGISOFT_URL = "http://download.agisoft.com/geoids"
+AGISOFT_URL = "https://s3-eu-west-1.amazonaws.com/download.agisoft.com/geoids"
```

Comment:
I seem to be getting access denied for this link:
![image](https://github.com/ACCESS-Cloud-Based-InSAR/dem-stitcher/assets/7882693/62975421-82f8-45c7-8706-c4dc8786c34e)

Reply:
Oh nevermind, I see the conversation in the issue now:
https://github.com/ACCESS-Cloud-Based-InSAR/dem-stitcher/issues/88#issuecomment-2197280311

Reply:
They don't have public `s3:ListBucket` permissions on that bucket. You'll get HTTP 403 Forbidden when trying to access any key that doesn't exist (rather than HTTP 404 Not Found). The resolved URLs to the data files are:
- https://s3-eu-west-1.amazonaws.com/download.agisoft.com/geoids/egm2008-1.tif
- https://s3-eu-west-1.amazonaws.com/download.agisoft.com/geoids/egm96-15.tif